src/main/protobuf/soc/gameState.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+
+package protos.soc;
+
+import "soc/board.proto";
+import "soc/inventory.proto";
+import "soc/playerState.proto";
+
+message GameState {
+    required Board board = 1;
+    repeated PlayerState playerStates = 2;
+    required int32 developmentCardsInDeck = 3;
+    required int32 currentTurn = 4;
```

Comment:
Is this the total turn count, or the current player who's turn it is, if the latter I would suggest, `current_turn_player` or `current_turn_player_position`

Reply:
will rename